### ISSUE TITLE
❓ Error Encountered with React Native Expo Sample App

### ISSUE THREAD

**Issue by SivanthP** [100mslive/100ms-react-native#1379]
Dear 100ms Team Support,

I hope this message finds you well. I'm writing to report an error that I encountered while using your React Native Expo sample app in conjunction with the React Native Modal package.

Upon integration and testing, I encountered the following error:

TypeError: _reactNative.DeviceEventEmitter.removeListener is not a function (it is undefined)

It seems that there is an issue related to the DeviceEventEmitter which is not recognized as a function, causing this TypeError. I believe this might be an issue with compatibility or configuration within the sample app.

Could you please provide guidance on how to resolve this error or if there are any updates or fixes available? Your assistance in resolving this matter would be greatly appreciated.

Thank you for your attention to this matter. I look forward to your response.

Best regards,
Sivanth Pachipala

**Comment by ygit** [comment]
@SivanthP were you able to resolve this issue?

**Comment by SivanthP** [comment]
No. If the change the code in node_modules it's working but when i tried to build the app in expo it is breaking over there. I am waiting for your support to fix it.

**Comment by ygit** [comment]
@SivanthP Let's get on a call to debug this. Please book a convenient time here: https://calendly.com/yogesh-singh-100ms/30min

**Comment by SivanthP** [comment]
Thank you guys. It's working fine even in the physical device. Thank you so much for your help. Have a nice day.

**Comment by ygit** [comment]
Awesome
